lib/index.js

```diff
@@ -16,17 +17,19 @@ module.exports = (pluginConfig, assetManager, options) => {
 };
 
 function buildConfig(pluginConfig, assetManager, { browsers, compact, sourcemaps }) {
-	return pluginConfig.map(bundleConfig => {
-		let inputFileName = assetManager.resolvePath(bundleConfig.source);
-		let target = assetManager.resolvePath(bundleConfig.target, {
-			enforceRelative: true
-		});
+	return pluginConfig.map(({ source, target, browserslist, fingerprint }) => {
+		if(!source || !target) {
+			let setting = source ? "target" : "source";
+			abort(`ERROR: missing ${repr(setting, false)} configuration`);
```

Comment:
Caveat: This doesn't state _which_ configuration (~bundle); might also be a JS one. Not sure how to succinctly address that - "in Sass bundle #1" perhaps?

Reply:
I would just put in "in a Sass bundle". I think that's sufficient to figure out the problem.